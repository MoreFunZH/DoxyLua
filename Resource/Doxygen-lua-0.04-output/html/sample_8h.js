var sample_8h =
[
    [ "_funcT", "sample_8h.html#aa58c7c5b374dcb87b81a4118dc2ad3c2", null ],
    [ "funcT", "sample_8h.html#af7234a096ca683a773efed4ea7c0d1d9", null ],
    [ "_gVar1", "sample_8h.html#ad4e9ac0922f761d2ab9eae3745ff4bc8", null ],
    [ "_gVar2", "sample_8h.html#a4c057e7b7f906d6f5762af2857dbfb99", null ],
    [ "sample1", "sample_8h.html#a5507b8cb084c3836378515f0cef1b923", null ]
];