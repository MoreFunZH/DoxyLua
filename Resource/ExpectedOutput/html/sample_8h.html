<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/xhtml;charset=UTF-8"/>
<meta http-equiv="X-UA-Compatible" content="IE=9"/>
<meta name="generator" content="Doxygen 1.8.6"/>
<title>Sample Project: sample.h File Reference</title>
<link href="tabs.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="jquery.js"></script>
<script type="text/javascript" src="dynsections.js"></script>
<link href="navtree.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="resize.js"></script>
<script type="text/javascript" src="navtree.js"></script>
<script type="text/javascript">
  $(document).ready(initResizable);
  $(window).load(resizeHeight);
</script>
<link href="search/search.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="search/search.js"></script>
<script type="text/javascript">
  $(document).ready(function() { searchBox.OnSelectItem(0); });
</script>
<link href="doxygen.css" rel="stylesheet" type="text/css" />
</head>
<body>
<div id="top"><!-- do not remove this div, it is closed by doxygen! -->
<div id="titlearea">
<table cellspacing="0" cellpadding="0">
 <tbody>
 <tr style="height: 56px;">
  <td style="padding-left: 0.5em;">
   <div id="projectname">Sample Project
   </div>
   <div id="projectbrief">This is the sample output of DoxyLua.</div>
  </td>
   <td>        <div id="MSearchBox" class="MSearchBoxInactive">
        <span class="left">
          <img id="MSearchSelect" src="search/mag_sel.png"
               onmouseover="return searchBox.OnSearchSelectShow()"
               onmouseout="return searchBox.OnSearchSelectHide()"
               alt=""/>
          <input type="text" id="MSearchField" value="Search" accesskey="S"
               onfocus="searchBox.OnSearchFieldFocus(true)" 
               onblur="searchBox.OnSearchFieldFocus(false)" 
               onkeyup="searchBox.OnSearchFieldChange(event)"/>
          </span><span class="right">
            <a id="MSearchClose" href="javascript:searchBox.CloseResultsWindow()"><img id="MSearchCloseImg" border="0" src="search/close.png" alt=""/></a>
          </span>
        </div>
</td>
 </tr>
 </tbody>
</table>
</div>
<!-- end header part -->
<!-- Generated by Doxygen 1.8.6 -->
<script type="text/javascript">
var searchBox = new SearchBox("searchBox", "search",false,'Search');
</script>
</div><!-- top -->
<div id="side-nav" class="ui-resizable side-nav-resizable">
  <div id="nav-tree">
    <div id="nav-tree-contents">
      <div id="nav-sync" class="sync"></div>
    </div>
  </div>
  <div id="splitbar" style="-moz-user-select:none;" 
       class="ui-resizable-handle">
  </div>
</div>
<script type="text/javascript">
$(document).ready(function(){initNavTree('sample_8h.html','');});
</script>
<div id="doc-content">
<!-- window showing the filter options -->
<div id="MSearchSelectWindow"
     onmouseover="return searchBox.OnSearchSelectShow()"
     onmouseout="return searchBox.OnSearchSelectHide()"
     onkeydown="return searchBox.OnSearchSelectKey(event)">
<a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(0)"><span class="SelectionMark">&#160;</span>All</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(1)"><span class="SelectionMark">&#160;</span>Files</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(2)"><span class="SelectionMark">&#160;</span>Functions</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(3)"><span class="SelectionMark">&#160;</span>Variables</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(4)"><span class="SelectionMark">&#160;</span>Enumerations</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(5)"><span class="SelectionMark">&#160;</span>Enumerator</a></div>

<!-- iframe showing the search results (closed by default) -->
<div id="MSearchResultsWindow">
<iframe src="javascript:void(0)" frameborder="0" 
        name="MSearchResults" id="MSearchResults">
</iframe>
</div>

<div class="header">
  <div class="summary">
<a href="#func-members">Functions</a> &#124;
<a href="#var-members">Variables</a>  </div>
  <div class="headertitle">
<div class="title">sample.h File Reference</div>  </div>
</div><!--header-->
<div class="contents">

<p>This is a sample lua file with comments.  
<a href="#details">More...</a></p>

<p><a href="sample_8h_source.html">Go to the source code of this file.</a></p>
<table class="memberdecls">
<tr class="heading"><td colspan="2"><h2 class="groupheader"><a name="func-members"></a>
Functions</h2></td></tr>
<tr class="memitem:aa58c7c5b374dcb87b81a4118dc2ad3c2"><td class="memItemLeft" align="right" valign="top">local function&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="sample_8h.html#aa58c7c5b374dcb87b81a4118dc2ad3c2">_funcT</a> (arg1, arg2)</td></tr>
<tr class="separator:aa58c7c5b374dcb87b81a4118dc2ad3c2"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:af7234a096ca683a773efed4ea7c0d1d9"><td class="memItemLeft" align="right" valign="top">function&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="sample_8h.html#af7234a096ca683a773efed4ea7c0d1d9">funcT</a> (arg1, arg2)</td></tr>
<tr class="separator:af7234a096ca683a773efed4ea7c0d1d9"><td class="memSeparator" colspan="2">&#160;</td></tr>
</table><table class="memberdecls">
<tr class="heading"><td colspan="2"><h2 class="groupheader"><a name="var-members"></a>
Variables</h2></td></tr>
<tr class="memitem:a5507b8cb084c3836378515f0cef1b923"><td class="memItemLeft" align="right" valign="top">local&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="sample_8h.html#a5507b8cb084c3836378515f0cef1b923">sample1</a> = require('sample1')</td></tr>
<tr class="separator:a5507b8cb084c3836378515f0cef1b923"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ad4e9ac0922f761d2ab9eae3745ff4bc8"><td class="memItemLeft" align="right" valign="top">local&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="sample_8h.html#ad4e9ac0922f761d2ab9eae3745ff4bc8">_gVar1</a> = 1</td></tr>
<tr class="memdesc:ad4e9ac0922f761d2ab9eae3745ff4bc8"><td class="mdescLeft">&#160;</td><td class="mdescRight">This is _gVar1.  <a href="#ad4e9ac0922f761d2ab9eae3745ff4bc8">More...</a><br/></td></tr>
<tr class="separator:ad4e9ac0922f761d2ab9eae3745ff4bc8"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a4c057e7b7f906d6f5762af2857dbfb99"><td class="memItemLeft" align="right" valign="top">local&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="sample_8h.html#a4c057e7b7f906d6f5762af2857dbfb99">_gVar2</a> = 2</td></tr>
<tr class="memdesc:a4c057e7b7f906d6f5762af2857dbfb99"><td class="mdescLeft">&#160;</td><td class="mdescRight">This is _gVar2.  <a href="#a4c057e7b7f906d6f5762af2857dbfb99">More...</a><br/></td></tr>
<tr class="separator:a4c057e7b7f906d6f5762af2857dbfb99"><td class="memSeparator" colspan="2">&#160;</td></tr>
</table>
<a name="details" id="details"></a><h2 class="groupheader">Detailed Description</h2>
<div class="textblock"><p>This is a sample lua file with comments. </p>
<dl class="section author"><dt>Author</dt><dd>dreamlover </dd></dl>

<p>Definition in file <a class="el" href="sample_8h_source.html">sample.h</a>.</p>
</div><h2 class="groupheader">Function Documentation</h2>
<a class="anchor" id="aa58c7c5b374dcb87b81a4118dc2ad3c2"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">local function _funcT </td>
          <td>(</td>
          <td class="paramtype">arg1&#160;</td>
          <td class="paramname">, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">arg2&#160;</td>
          <td class="paramname">&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>This is _funcT. </p>
<dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">arg1</td><td>This is arg1. </td></tr>
    <tr><td class="paramname">arg2</td><td>This is arg2. </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>Return something. </dd></dl>

</div>
</div>
<a class="anchor" id="af7234a096ca683a773efed4ea7c0d1d9"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">function funcT </td>
          <td>(</td>
          <td class="paramtype">arg1&#160;</td>
          <td class="paramname">, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">arg2&#160;</td>
          <td class="paramname">&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>This is funcT. </p>
<dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">arg1</td><td>This is arg1. </td></tr>
    <tr><td class="paramname">arg2</td><td>This is arg2. </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>Return something. </dd></dl>

</div>
</div>
<h2 class="groupheader">Variable Documentation</h2>
<a class="anchor" id="ad4e9ac0922f761d2ab9eae3745ff4bc8"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">local _gVar1 = 1</td>
        </tr>
      </table>
</div><div class="memdoc">

<p>This is _gVar1. </p>

<p>Definition at line <a class="el" href="sample_8h_source.html#l00011">11</a> of file <a class="el" href="sample_8h_source.html">sample.h</a>.</p>

</div>
</div>
<a class="anchor" id="a4c057e7b7f906d6f5762af2857dbfb99"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">local _gVar2 = 2</td>
        </tr>
      </table>
</div><div class="memdoc">

<p>This is _gVar2. </p>

<p>Definition at line <a class="el" href="sample_8h_source.html#l00013">13</a> of file <a class="el" href="sample_8h_source.html">sample.h</a>.</p>

</div>
</div>
<a class="anchor" id="a5507b8cb084c3836378515f0cef1b923"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">local sample1 = require('sample1')</td>
        </tr>
      </table>
</div><div class="memdoc">

<p>Definition at line <a class="el" href="sample_8h_source.html#l00008">8</a> of file <a class="el" href="sample_8h_source.html">sample.h</a>.</p>

</div>
</div>
</div><!-- contents -->
</div><!-- doc-content -->
<!-- start footer part -->
<div id="nav-path" class="navpath"><!-- id is needed for treeview function! -->
  <ul>
    <li class="navelem"><a class="el" href="sample_8h.html">sample.h</a></li>
    <li class="footer">Generated on Mon Mar 31 2014 11:18:48 for Sample Project by
    <a href="http://www.doxygen.org/index.html">
    <img class="footer" src="doxygen.png" alt="doxygen"/></a> 1.8.6 </li>
  </ul>
</div>
</body>
</html>
